tests/kokkos-based/helpers.hpp

```diff
@@ -92,7 +92,7 @@ mdspan_t make_mdspan(ValueType *data, std::size_t ext0, std::size_t ext1) {
 template<class A_t, class ValueType = typename A_t::value_type>
 void set(A_t A, ValueType value)
 {
-  using size_type = typename std::experimental::extents<>::size_type;
+  using size_type = typename Kokkos::extents<size_t>::size_type;
```

Comment:
`size_type` is `make_unsigned_t<index_type>`.  You probably want `index_type` throughout.